### PR TITLE
Updated docs with community section

### PR DESCRIPTION
**Pull request by niksirbi**
## The Community section
I updated the movement docs to add a new "Community" section.
It could also be called "About", but I prefer the "Community" term (borrowed from [napari docs](https://napari.org/stable/community/index.html)).

This section includes:
- A community landing page, with links to the parts below and contact info
- the statement on mission, scope, and design principles.
- the roadmap
- the contributing guide (it was a standalone page before, now I have moved it within this section)
- Related projects
- License (included from the file in the repository)

Many thanks to @adamltyson, @lochhh and @sfmig for helping me draft the mission, scope, design principles and roadmap!

## Accompanying changes
The docs landing page and the repo README have also been updated to reflect the new structure of the docs. There is an "Overview" section in both, which contains the text @adamltyson wrote for the [neuroinformatics main website](https://neuroinformatics.dev/projects.html#projects-behaviour). There is some regrettable repetition between these two documents (violating DRY), but including content from one of them to the other creates some nasty formatting issues (due to syntax differences between GitHub and Sphinx-Myst markdown). I think some repetition is fine for now, unless we want to write a GitHub action for building the README file automatically. Some links in the README will not work because they lead to the new website pages (which have not been deployed yet), but this should be fixed after merging and releasing.

I also was tempted to include some small docs fixes, e.g. changing the copyright holder to UCL, and making the URL for some pages more sensible.

## What is missing from the Community section
- A section about the core project members? Similar to [this napari page](https://napari.org/stable/community/team.html). It should also mention NIU, SWC, etc.
- A funding section
- A contributors page, thanking everyone, similar to the [brainglobe one](https://brainglobe.info/contributors.html)
- A governance page (we don't absolutely need one now, but we may need it soon)
- The "Related projects" section needs to be fleshed out as per #58 

## How to review this PR
The best way is to build the docs website locally, following [these instructions](https://movement.neuroinformatics.dev/contributing.html#building-the-documentation-locally).

### CODE REVIEW

File: docs/source/community/mission-scope.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## Mission
+
+[movement](https://movement.neuroinformatics.dev/) aims to **facilitate the study of animal behaviour in neuroscience** by providing a suite of **Python tools to analyse body movements** across space and time.
+
+## Scope
+
+At its core, movement handles trajectories of *keypoints*, which are specific body parts of an *individual*. An individual's posture or *pose* is represented by a set of keypoint coordinates, given in 2D (x,y) or 3D (x,y,z). The sequential collection of poses over time forms *pose tracks*. In neuroscience, these tracks are typically extracted from video data using software like [DeepLabCut](http://www.mackenziemathislab.org/deeplabcut) or [SLEAP](https://sleap.ai/).
+
+With movement, our vision is to present a **consistent interface for pose tracks** and to **analyze them using modular and accessible tools**. We aim to accommodate data from a range of pose estimation packages, in **2D or 3D**, tracking a **single or multiple individuals**. The focus will be on providing functionalities for data cleaning, visualisation and motion quantification (see the [Roadmap](target-roadmap) for details).
```
Comment (adamltyson) [inline docs/source/community/mission-scope.md:12]:
```suggestion
With movement, our vision is to present a **consistent interface for pose tracks** and to **analyze them using modular and accessible tools**. We aim to accommodate data from a range of pose estimation packages, in **2D or 3D**, tracking **single or multiple individuals**. The focus will be on providing functionalities for data cleaning, visualisation and motion quantification (see the [Roadmap](target-roadmap) for details).
```

**Review by adamltyson — APPROVED** [review]
LGTM! 🎉

File: docs/source/community/mission-scope.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+movement is committed to:
+- __Ease of installation and use__. We aim for a cross-platform installation and are mindful of dependencies that may compromise this goal.
+- __User accessibility__, catering to varying coding expertise by offering both a GUI and a Python API.
+- __Comprehensive documentation__, enriched with tutorials and examples.
+- __Robustness and maintainability__ through high test coverage.
+- __Scientific accuracy and reproducibility__ by validating inputs and outputs.
+- __Performance and responsiveness__, especially for large datasets, using parallel processing where appropriate.
+- __Modularity and flexibility__. We envision movement as a platform for new tools and analyses, offering users the building blocks to craft their own workflows.
+
+Some of these principles are shared with, and were inspired by, napari's [Mission and Values](https://napari.org/mission.html) statement.
```
Comment (lochhh) [inline docs/source/community/mission-scope.md:27]:
```suggestion
Some of these principles are shared with, and were inspired by, napari's [Mission and Values](https://napari.org/stable/community/mission_and_values.html) statement.
```

File: docs/source/community/roadmap.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+(target-roadmap)=
+# Roadmap
+
+The roadmap outlines **current development priorities** and aims to **guide core developers** and to **encourage community contributions**. It is a living document and will be updated as the project evolves.
+
+The roadmap is **not meant to limit** movement features, as we're open to suggestions and contributions. Join our [Zulip chat](https://neuroinformatics.zulipchat.com/#narrow/stream/406001-Movement/topic/Welcome!) to share your ideas. We'll take community demand and feedback into account when planning future releases.
```
Comment (lochhh) [inline docs/source/community/roadmap.md:6]:
```suggestion
The roadmap is **not meant to limit** movement features, as we are open to suggestions and contributions. Join our [Zulip chat](https://neuroinformatics.zulipchat.com/#narrow/stream/406001-Movement/topic/Welcome!) to share your ideas. We will take community demand and feedback into account when planning future releases.
```

File: docs/source/community/roadmap.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+- __Clean pose tracks__, including, but not limited to, handling of missing values, filtering, smoothing, and resampling.
+- __Derive kinematic variables__ like velocity, acceleration, joint angles, etc., focusing on those prevalent in neuroscience.
+- __Integrate spatial data about the animal's environment__ for combined analysis with pose tracks. This covers regions of interest (ROIs) such as the arena in which the animal is moving and the location of objects within it.
+- __Define and transform coordinate systems__. Coordinates can be relative to the camera, environment, or the animal itself (egocentric).
+
+## Short-term milestone - `v0.1`
+We plan to release version `v0.1` of movement in early 2024, providing a minimal set of features to demonstrate the project's potential and to gather feedback from users. At minimum, it should include the following features:
+
+- Importing pose tracks from [DeepLabCut](http://www.mackenziemathislab.org/deeplabcut) and [SLEAP](https://sleap.ai/) into a common `xarray.Dataset` structure. This has been largely accomplished, but some remaining work is required to handle special cases.
```
Comment (lochhh) [inline docs/source/community/roadmap.md:22]:
```suggestion
- Importing pose tracks from [DeepLabCut](https://www.mackenziemathislab.org/deeplabcut) and [SLEAP](https://sleap.ai/) into a common `xarray.Dataset` structure. This has been largely accomplished, but some remaining work is required to handle special cases.
```

File: docs/source/conf.py [hunk 5]
```diff
@@ -21,7 +21,7 @@
 sys.path.insert(0, os.path.abspath("../.."))
 
 project = "movement"
-copyright = "2022, UCL"
+copyright = "2023, University College London"
```
Comment (lochhh) [inline docs/source/conf.py:24]: Do you want to do something like:

```python
from datetime import date

copyright = f"2022–{date.today().year}, University College London"
```

File: docs/source/index.md [hunk 6]
```diff
@@ -13,52 +13,38 @@ Install and try it out.
...
-
-## Status
-:::{warning}
-- 🏗️ The package is currently in early development. Stay tuned ⌛
-- It is not sufficiently tested to be used for scientific analysis
-- The interface is subject to changes
-:::
+## Overview
 
+Pose estimation tools, such as [DeepLabCut](http://www.mackenziemathislab.org/deeplabcut) and [SLEAP](https://sleap.ai/) are now commonplace when processing video data of animal behaviour. There is not yet a standardised, easy to use way to process the *pose tracks* produced from these software packages.
```
Comment (lochhh) [inline docs/source/index.md:32]:
```suggestion
Pose estimation tools, such as [DeepLabCut](http://www.mackenziemathislab.org/deeplabcut) and [SLEAP](https://sleap.ai/) are now commonplace when processing video data of animal behaviour. There is not yet a standardised, easy-to-use way to process the *pose tracks* produced from these software packages.
```

File: docs/source/snippets/status-warning.md [hunk 7]
```diff
@@ -0,0 +1,6 @@
+:::{admonition} Status
+:class: warning
+- 🏗️ The package is currently in early development. Stay tuned ⌛
+- It is not sufficiently tested to be used for scientific analysis
+- The interface is subject to changes
```
Comment (lochhh) [inline docs/source/snippets/status-warning.md:5]:
```suggestion
- It is not sufficiently tested to be used for scientific analysis.
- The interface is subject to changes.
```

**Review by lochhh — APPROVED** [review]
Great work! 🚀

File: docs/source/conf.py (hunk 5 above)
Comment (niksirbi) [reply docs/source/conf.py:24]: I'm actually not sure how copyright years work, is it supposed to be the date of creation, or always the current date? Also how does it work with versioned products, like software? Tagging our resident IP specialist @adamltyson 😉

Comment (niksirbi) [reply docs/source/conf.py:24]: After some digging, specifying a year range should be appropriate, so you were right @lochhh

Comment (adamltyson) [reply docs/source/conf.py]: I usually just add the year the software was first released. IIRC copyright is automatic in the UK, so it doesn't really make much difference anyway. I would just set it to be 2023 and leave it for simplicity.

File: docs/source/conf.py [hunk 8]
```diff
@@ -21,7 +22,8 @@
 sys.path.insert(0, os.path.abspath("../.."))
 
 project = "movement"
-copyright = "2023, University College London"
+current_year = date.today().year
+copyright = f"2022-{current_year}, University College London"
```
Comment (adamltyson) [inline docs/source/conf.py:26]: As commented above. I would leave this as 2023 everywhere. Otherwise the docs could get out of date with the LICENSE.

**Review by adamltyson — APPROVED** [review]

**Review by lochhh — APPROVED** [review]

**Comment by niksirbi** [comment]
This error in building the docs took me forever to track down, but I now know its source.
Strap in, because it's a loong story:

Because our documentation includes examples built with [Sphinx-Gallery](https://sphinx-gallery.github.io/stable/index.html), `movement` itself has to be included in `docs/requirements.txt`. This is achieved by adding `-e .` at the top of that file. So far this worked fine.

However, `movement` depends on [`sleap-io`](https://github.com/talmolab/sleap-io), which in turn depends on [PyAV](https://github.com/PyAV-Org/PyAV) - a package providing Python bindings for `ffmpeg`. PyAV, which is available on pip as `av`, [doesn't (yet) provide wheels for Python3.12](https://github.com/PyAV-Org/PyAV/issues/1168).

Normally, this wouldn't be a problem for us, since we officially support Python versions 3.9 - 3.11. That said, the [reusable workflow we use for building the Sphinx docs](https://github.com/neuroinformatics-unit/actions/blob/main/build_sphinx_docs/action.yml) includes:
```yaml
  - name: Setup Python
    uses: actions/setup-python@v4
    with:
      python-version: '3.x'
```
`3.x` means the latest stable version, and must have recently been changed to 3.12. So when this action runs, it tries to install `movement` -> `sleap-io` -> `av` in a Python 3.12 environment, and fails to find the wheels. I reproduced this locally on Ubuntu. The documentation build would fail on 3.12 and succeed on all previous versions.

We could solve this in a number of ways:
- Modify the reusable Sphinx build action to use Python 3.11 (instead of the latest). This is probably the easiest option. Or even better, expose a Python version option for this workflow, so it can be flexibly chosen depending on the package.
- Ask `sleap-io` developers to switch their dependency from `PyAV` to [`pyav`](https://github.com/WyattBlue/pyav) - which is a recent fork and provides the right wheels. But this may be unnecessary and rushed, if we assume that PyAV will eventually add the wheels.

**Comment by niksirbi** [comment]
I solved the issue by updating the GitHub action and running it with Python 3.11. Thanks @alessandrofelder for approving the action update. Merging now.

**Pull request merged**